content/develop/concepts/configuration/theming-colors-and-borders.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Colors and borders
+slug: /develop/concepts/configuration/theming-customize-colors-and-borders
+---
+
+# Customize colors and borders in your Streamlit app
+
+## Color values
+
+For all configuration options that accept a color, you can specify the value with one of the following strings:
+
+- A CSS [`<named-color>`](https://developer.mozilla.org/en-US/docs/Web/CSS/named-color) like `"darkblue"` or `"maroon"`.
+- A HEX string like `"#483d8b"` or `"#6A5ACD"`.
+- An RGB string like `"rgb(106, 90, 205)"` or `"RGB(70, 130, 180)"`.
+- An HSL string like `"hsl(248, 53%, 58%)"` or `"HSL(147, 50%, 47%)"`.
+
+<Tip>
+
+While it is possible for you to specify an alpha value for your colors, this isn't recommended. Streamlit adjusts the alpha value of colors to ensure contextually appropriate shading between background and foreground.
+
+</Tip>
+
+## Default Streamlit colors
+
+Streamlit comes with two preconfigured themes: light and dark. If you don't specify a theme, for each user, Streamlit will attempt to use the preconfigured theme that best matches their browser settings.
```

Comment:
Might be cool to add an image of the default light/dark theme below this paragraph.